src/stream-ciphers.rst

```diff
@@ -255,83 +255,81 @@ Conclusion
 ^^^^^^^^^^
 
 In the real world, block ciphers are used in systems that encrypt large
-amounts of data all the time. We've seen that when using :term:`ECB mode`, an
-attacker can both analyze ciphertexts to recognize repeating patterns,
-and even decrypt messages when given access to an :term:`encryption oracle`.
+amounts of data all the time. We see that when using :term:`ECB mode`, an
+attacker both analyzes ciphertexts to recognize repeating patterns,
+and even decrypts messages when given access to an :term:`encryption oracle`.
 
 Even when we use idealized block ciphers with unrealistic properties,
-such as block sizes of more than a thousand bits, an attacker ends up
-being able to decrypt the ciphertexts. Real world block ciphers only
-have more limitations than our idealized examples, such as much smaller
+such as block sizes of more than a thousand bits, an attacker can
+decrypt the ciphertexts. Real world block ciphers
+have more limitations than our idealized examples, for example, having much smaller
 block sizes.
 
-We aren't even taking into account any potential weaknesses in the block
-cipher. It's not AES (or our test block ciphers) that cause this
-problem, it's our ECB construction. Clearly, we need something better.
+We are not yet even considering potential weaknesses in the block
+cipher. It is not AES nor the test block ciphers that cause the
+problem, it is our ECB construction. Clearly, something better is needed.
 
 Block cipher modes of operation
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-One of the more common ways of producing a :term:`stream cipher` is to use a
+A common way of producing a :term:`stream cipher` is by using a
 block cipher in a particular configuration. The compound system behaves
-like a :term:`stream cipher`. These configurations are commonly called
-:term:`mode of operation`\s. They aren't specific to a particular block cipher.
+like a :term:`stream cipher`. These configurations are known as
+:term:`mode of operation`\s. They are not specific to a particular block cipher.
```

Comment:
@lvh ,

"They are not specific..."
Can this change to note that different modes are available?

Reply:
I'm not sure i understand, can you elaborate?

Reply:
"They are not specific to a particular block cipher."
Are you trying to say different modes of operations can be applied to block ciphers other than AES?

Reply:
I'm trying to say that modes of operation (I don't count e.g. AEZ as a real mode of operation) are defined independently of block ciphers. You can do AES-CBC, but also 3DES-CBC. I'm trying to say all modes of operation work on all block ciphers, with very few edge cases.

Reply:
Yup that's how I understood it

What I didn't know is if the beginning word "They" is correctly representing `mode of operations` or "These configurations" (from previous sentence)

Reply:
Hm, gotcha. The previous sentence just defines the name, so since "configurations" and "modes of operation" are the same thing I guess I don't see it as confusing, but if you want to replace the `They` with "modes of operation" that's fine

Reply:
That makes sense. I don't think it needs to be changed. Thanks for explaining.